ros2node/ros2node/verb/info.py

```diff
@@ -11,6 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import sys
```

Comment:
You should have a blank line between the first import and the copyright/license header, as it was before.

```suggestion

import sys
```

Reply:
I tried to fix this up myself, but it won't let me push to your branch.

Reply:
Yeah it's on our dev fork. Added the commit. Should we update ament-copyright to catch this particular nit so you don't have to be the human spellcheck? =P

Reply:
To be honest, I thought that PEP8 (flake8) would catch it.

Reply:
It seems we are not enforcing it, and as best I can tell it's not one of the ones we're disabling in ament_flake8.ini, most of those are docstring-related

Reply:
It's probably a corner case, since flake8 assumes it's a comment about the import statement, which it isn't, but it cannot know that. I guess the copyright checker could assert an empty line after the statement.